### ISSUE TITLE
Dropout layers should change from tf.layers.dropout to tf.nn.dropout

### ISSUE THREAD

**Issue by aiida-** [USC-CSSL/NTAP#51]
tf.layer.dropout needs a third argument (training=True) to be activated. 
Change it to tf.nn.dropout in order for it to work correctly.
